routes/teams.js

```diff
@@ -23,4 +24,101 @@ router.get('/team', function(req, res) {
     });
 });
 
+
+// when players endpoint is hit, call the API/DB using that team_id
+router.get('/players', function(req, res) {
+  temp = req.query.split('=');
```

Comment:
because the query comes back as "team_id=1234567"... and I only want the id, not the label.